### PR TITLE
Document AWS SSO setup

### PR DESCRIPTION
**Pull request by yuvipanda**

### CODE REVIEW

File: practices/cloud-access.md [hunk 1]
```diff
@@ -34,9 +34,28 @@ The canonical list of GCP projects we have access to is maintained
...
-in every AWS account we have access to. This should have the broadest
+We have two ways to access AWS accounts.
+
+### AWS SSO
+
+For accounts we fully control, we use [AWS SSO](https://aws.amazon.com/single-sign-on/)
+to access them. We have an [AWS organization](https://aws.amazon.com/organizations/)
+set up, and sub-accounts for reach AWS account. This lets us use just
+*one* set of user credentials for multiple AWS accounts, with
+separate billing and access control. You can log-in at
```
Comment (GeorgianaElena) [inline practices/cloud-access.md:45]: Note for @2i2c-org/tech-team: before login, you first need to accept the AWS Single Sign-On invitation on email :sweat_smile:

**Review by GeorgianaElena — APPROVED** [review]
Thanks for documenting this @yuvipanda! I managed to access the console :tada:

**Review by sgibson91 — APPROVED** [review]
I also managed to login after accepting the email invite 🙌🏻

File: practices/cloud-access.md [hunk 2]
```diff
@@ -34,9 +34,28 @@ The canonical list of GCP projects we have access to is maintained
...
 
-Each 2i2c engineer should get an individual [IAM User Account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html)
-in every AWS account we have access to. This should have the broadest
+We have two ways to access AWS accounts.
+
+### AWS SSO
+
+For accounts we fully control, we use [AWS SSO](https://aws.amazon.com/single-sign-on/)
+to access them. We have an [AWS organization](https://aws.amazon.com/organizations/)
+set up, and sub-accounts for reach AWS account. This lets us use just
```
Comment (choldgraf) [inline practices/cloud-access.md:43]:
```suggestion
set up, and sub-accounts for each AWS account. This lets us use just
```

File: practices/cloud-access.md [hunk 3]
```diff
@@ -34,9 +34,28 @@ The canonical list of GCP projects we have access to is maintained
...
+*one* set of user credentials for multiple AWS accounts, with
```
Comment (choldgraf) [inline practices/cloud-access.md:44]: Does this mean that each of us can have multiple AWS accounts with a single log-in portal, or that 2i2c can have us all log into the account using the same credentials?

**Review by choldgraf — APPROVED** [review]
Just some minor comments - thanks a lot for writing this up!

**Review by choldgraf — APPROVED** [review]
Let's merge this in and we can iterate on the language if it is unclear to folks in the future. Thanks @yuvipanda !

**Comment by damianavila** [comment]
@yuvipanda, the invitation you sent me expired whereas I was away, can you (or someone else) send it again? Thanks!

**Comment by yuvipanda** [comment]
@damianavila sent you a reset password email!

**Comment by damianavila** [comment]
I tried but it failed, maybe because I tried after 24 hs??
Can you send me another one and ping me on Slack about it so I do it right away?
Sorry for the bother...

**Comment by damianavila** [comment]
Just for completeness, Yuvi resent me the invitation and I was able to validate it.

**Pull request merged**